Doc/library/xml.sax.utils.rst

```diff
@@ -25,6 +25,11 @@ or as base classes.
    replaced with its corresponding value.  The characters ``'&'``, ``'<'`` and
    ``'>'`` are always escaped, even if *entities* is provided.
 
+   .. note::
+
+      This function should only be used to escape characters that are
+      problematic to include directly in XML. Do not use this function as a general
```

Comment:
This wording doesnt seem too ideal for me, can we use serhey's `that can't be used directly in XML ...`.